lops/lop-domain-r5-zynqmp-rpmsg-kernelspace.dts

```diff
@@ -90,7 +90,7 @@
                   };
                   lop_14 {
                          compatible = "system-device-tree-v1,lop,output";
-                         outfile = "linux.dts";
+                         outfile = "linux.dtb";
```

Comment:
The question is .. why ? Is there a bug with the dts output ?

Reply:
as mentioned earlier, just to comply with convention. they do compile